### PR TITLE
KAZOO-3103: topup teletype template

### PR DESCRIPTION
**Pull request by fenollp**

### CODE REVIEW

**Comment by fenollp** [comment]
Note: this adds things in util that upcoming KAZOO-3100 will depend on.

File: applications/teletype/src/teletype_util.erl [hunk 1]
```diff
@@ -308,29 +313,24 @@ account_params(DataJObj) ->
 
 -spec find_account_params(wh_json:object(), ne_binary()) -> wh_proplist().
 find_account_params(DataJObj, AccountId) ->
-    case teletype_util:open_doc(<<"account">>, AccountId, DataJObj) of
-        {'ok', AccountJObj} -> build_account_params(AccountJObj);
+    case ?MODULE:open_doc(<<"account">>, AccountId, DataJObj) of
```
Comment (lazedo) [inline applications/teletype/src/teletype_util.erl]: you changed teletype_util:open_doc to ?MODULE:open_doc , what does this solve ?
why not change to open_doc ? if we move open_doc outside this module we will get compilation error, which i believe is better then getting the errors when running.

File: applications/teletype/src/teletype_shared_listener.erl [hunk 2]
```diff
@@ -55,12 +55,13 @@
                       }
                      ,{{'teletype_system_alert', 'handle_system_alert'}
                        ,[{<<"notification">>, <<"system_alert">>}]
+                     ,{{'teletype_topup', 'handle_topup'}
```
Comment (lazedo) [inline applications/teletype/src/teletype_shared_listener.erl]: missing } before the new entry ?

Comment (fenollp) [inline applications/teletype/src/teletype_shared_listener.erl]: Ah. Rebase failed me

File: applications/teletype/src/teletype_util.erl (hunk 1 above)
Comment (fenollp) [inline applications/teletype/src/teletype_util.erl]: You're very right

**Comment by fenollp** [comment]
Ready for merging!

**Pull request merged**